ajax/libs/mathjax-mhchem/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "mathjax-mhchem",
+  "filename": "mhchem.js",
```

Comment:
`"filename"` should point to the main minified file. It would be `mhchem.min.js`. Thanks.